src/controllers/v1.controller.ts

```diff
@@ -136,7 +137,7 @@ export class V1Controller {
         return this.loadBalancerRelay(redirect.loadBalancerID, rawData)
       }
     }
-    return new HttpErrors.InternalServerError('Invalid domain')
+    return jsonrpc.error(1, new jsonrpc.JsonRpcError('Invalid domain', -32000)) as ErrorObject
```

Comment:
why you use `jsonrpc.JsonRpcError` here and `ErrorObject` in other places, in what they differ?

Reply:
jsonrpc.JsonRpcError 
```
error: jsonrpc.JsonRpcError { 
message: string,
code: number
}
```

ErrorObject
```
jsonrpc: string,
id: number,
error: jsonrpc.JsonRpcError { 
message: string,
code: number
}
```

Reply:
when one has to be used over the other?

Reply:
> 

No preference, both do the same — I'll just use one of them for consistency. The `jsonrpc.error()` is preferred.

Reply:
> 

Yes